custom_components/healthchecksio/__init__.py

```
@@ -4,6 +4,13 @@
 For more details about this component, please refer to
 https://github.com/custom-components/healthchecksio
 """
+from __future__ import annotations
+
+from logging import getLogger
```

Ruff (0.11.9)

`logging.getLogger` imported but unused

(F401)